### PR TITLE
sim: make run at fixed scale an option

### PR DESCRIPTION
**Pull request by JSwambo**

### CODE REVIEW

File: Model/main.py [hunk 1]
```diff
@@ -27,6 +27,8 @@
 INVALID_SPEND_RATE = os.getenv("INVALID_SPEND_RATE", None)
 # Catastrophe rate per day
 CATASTROPHE_RATE = os.getenv("CATASTROPHE_RATE", None)
+# Delegate rate per day (if scale_fixed)
+DELEGATE_RATE = os.getenv("DELEGATE_REATE", None)
```
Comment (darosior) [inline Model/main.py:31]: s/REATE/RATE/

File: Model/results.py [hunk 2]
```diff
@@ -126,21 +129,22 @@ def multiprocess_run(range_seed, val, study_type, config_map):
...
         "CATASTROPHE_RATE": 0.005,
-        "CANCEL_COIN_SELECTION": 0,
+        "DELEGATE_RATE": 1,
     }
 
     # Set the study parameters
-    study_type = "O_VERSION"
-    val_range = [0, 1]
+    study_type = "REFILL_EXCESS"
+    val_range = [0, 0.5, 1, 1.5, 2]
```
Comment (darosior) [inline Model/results.py:147]: Unrelated change? But nvm

**Review by darosior — COMMENTED** [review]
I don't understand the purpose of this.

**Comment by JSwambo** [comment]
> I don't understand the purpose of this.

Please see discussion in the paper early in section 3.4. Also the results for figure 4.

**Pull request closed without merging**